packages/event-cache/src/EventCache.js

```diff
@@ -15,6 +15,18 @@ const {
 } = require('./backends/browser')
 
 const limiter = new Bottleneck({ maxConcurrent: 25 })
+limiter.on('error', err => {
+  console.log('Error occurred within rate limiter', err)
```

Comment:
can we use debug rather than console.log ?

Reply:
doh!  Thanks